xxx/build.gradle

```diff
@@ -21,8 +20,8 @@ repositories {
 }
 
 dependencies {
-    compile('org.springframework.boot:spring-boot-starter-webflux')
-    testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile("org.springframework.boot:spring-boot-starter-webflux:${springBootVersion}")
+    testCompile("org.springframework.boot:spring-boot-starter-test:${springBootVersion}")
```

Comment:
`dependency-management` plugin do not fill versions without `spring boot` plugin so I have to define it explicitly.